style.css

```diff
@@ -0,0 +1,461 @@
+  display: flex;
+}
+
+header .nav-menu ul li {
+  list-style: none;
+  padding-left: 30px;
+}
+
+header .nav-menu ul li a {
+  text-decoration: none;
+  color: #f8f8f8;
+  font-size: 18px;
+  font-weight: 600;
+  transition: 0.5s;
+}
+
+header .nav-menu .nav-button {
+  display: none;
+  color: #f8f8f8;
+}
+
+.headline .social-network ul li a {
+  text-decoration: none;
+  color: #f8f8f8;
+}
+
+header .nav-menu ul li a:hover {
+  padding-bottom: 10px;
+  border-bottom: solid 3px #ff6b00;
+}
+
+/* ====== Headline ====== */
+
+#headline {
+  background-color: #1c1a19;
+}
+
+.headline {
+  display: flex;
+  align-items: center;
+  height: calc(100vh - 65px);
+  max-height: 700px;
+  color: #f8f8f8;
+  background-size: 650px 650px;
+  background-repeat: no-repeat;
+  background-position: 55vw 0;
+  background-image: url('./images/Illustration-header.png');
+}
+
+.headline .social-network ul {
+  padding-right: 40px;
+}
+
+.headline .social-network ul li {
+  list-style: none;
+  width: 25px;
+  height: 25px;
+  overflow: hidden;
+  margin-top: 10px;
+  transition: 0.5s;
+}
+
+.about-me .competence .competence-list li {
+  list-style: none;
+  font-family: 'Inter', sans-serif;
+  font-style: normal;
+  font-weight: 400;
+  font-size: 22px;
+  line-height: 43px;
+}
+
+.headline .social-network ul li:hover {
+  transform: scale(1.2);
+}
+
+.headline .resume {
+  flex-basis: 40%;
+  padding: 0 30px;
+}
+
+.my-work h1 {
+  font-family: 'Crete Round', sans-serif;
+  font-weight: 400;
+  font-size: 40px;
+  line-height: 52px;
+  color: #091e42;
+}
+
+.headline .resume h1 {
+  font-size: 3rem;
+  padding: 0 !important;
+  margin: 0 !important;
+  font-family: 'Crete Round', sans-serif;
+  font-weight: 400;
+  color: #ff6b00;
+}
+
+.headline .resume h4 {
+  font-size: 1.5rem;
+  font-family: Inter, sans-serif;
+  font-weight: 500;
+}
+
+.headline .resume p {
+  font-family: 'Poppins', sans-serif;
+  font-style: normal;
+  font-weight: 400;
+  font-size: 18px;
+  line-height: 24px;
+  max-width: 566px;
+}
+
+/* --------- Works ---------- */
+.my-work {
+  padding: 100px 100px;
+}
+
+.my-work .grid {
+  display: grid;
+  grid-template-columns: 1fr 1fr 1fr;
+  grid-gap: 2rem;
+}
+
+.my-work .cards .card {
+  width: 369px;
+  height: 435px;
+  display: flex;
+  position: relative;
+  flex-direction: column;
+  justify-content: flex-end;
+  color: #f8f8f8;
+  margin-left: auto;
+  margin-right: auto;
+  overflow: hidden;
+  background-image: url('https://cdn.shopify.com/s/files/1/0430/1861/6996/products/lunette-rudy-project-deltabeat-orange-cote.jpg?v=1634910781');
+  background-color: #f58931;
+  background-size: 100% 100%;
+}
+
+.my-work .cards .card-featured {
+  grid-column: 1/-1;
+  height: 435px;
+  display: flex;
+  position: relative;
+  justify-content: flex-end;
+  color: #091e42;
+  margin-left: auto;
+  margin-right: auto;
+  overflow: hidden;
+  background-size: 100% 100%;
+}
+
+.my-work .cards .format p {
+  font-family: 'Inter', sans-serif;
```

Comment:
- [ ] [OPTIONAL] For increasing 🆙 your chances in the job search 💼 after finishing microverse, we request that you follow coding best practices 💯  in all of your projects. One of these practices is **DRY** (**D**on't **R**epeact **Y**ourself). That means if you find yourself repeating 🔁 a specific line of code in multiple places, you have to find a way to only use it once 1️⃣. This way when you update ♻️  this line, you won't have to remember where else did you write that to update the other copies.

### You can check this in the [Html & Css Best Practices Page](https://github.com/microverseinc/curriculum-html-css/blob/main/articles/html_css_best_practices.md)

![image](https://user-images.githubusercontent.com/60979926/164267978-27c9c5e1-37bb-4d2d-81ef-e7cd491bfbdb.png)

### In our case

you can easily use a `.font` class in multiple places or simply put the `font-family` property in the `body` selector and all other children elements going to inherit it by default.

- (your code)

```css
.my-work .cards .format p {
  /* ...code */
  font-family: 'Inter', sans-serif;
  /* ...code */
}

.my-work .cards .card-featured.format .categories li {
  /* ...code */
  font-family: 'Inter', sans-serif;
  /* ...code */
}
```

- suggestion

```css
.font {
  font-family: 'Inter', sans-serif;
}
```

> **Note**: Also, you can do the same thing with `display: flex` declaration.